features/order/managing_orders/seeing_order_item_detailed_data.feature

```diff
@@ -8,27 +8,30 @@ Feature: Seeing order item detailed data
         Given the store operates on a single channel in "United States"
         And the store has "VAT" tax rate of 10% for "T-Shirts" within the "US" zone
         And the store classifies its products as "T-Shirts"
-        And the store has a product "Iron Man T-Shirt" priced at "$39.00"
-        And it belongs to "T-Shirts" tax category
+        And the store has a product "Marvel T-Shirt"
+        And this product has "Iron Man T-Shirt" variant priced at "$49.00"
+        And this product has "Thor T-Shirt" variant priced at "$39.00"
```

Comment:
Not in scope of this PR.
Besides most of our steps need a hardcore semantical rework